README.md

```diff
@@ -103,6 +103,41 @@ Phoenix application named `MyApp`.
     When using the wrapper module's `my_custom_operation`, the time it takes to call
     `MyApp.Repo.my_custom_operation/2` will be recorded to New Relic.
 
+## Error Reporting
+   If you want to report all errors to new relic
+
+1.  Configure your router
+    ```elixir
+    # lib/my_app/router.ex
+
+    defmodule MyApp.Router do
+      use MyApp, :router
+      use NewRelixir.Plug.Exception
+
+      ...
+    end
+    ```
+
+    If you want to report caught errors
+
+    ```elixir
+    try do
+      raise "oops"
+    catch
+      kind, reason ->
```

Comment:
🤔 I'm a bit confused or I just have a lack of understanding here, but should the `catch` include a match for the error raised? 

I also looked up `NewRelixir.CurrentTransaction.get/0` but didn't find anything. Perhaps something as simple as a string like `"Update user account"` would suffice? Without looking at the interface for `record_error`, I couldn't tell that `transaction` was actually `transaction_name` implying that it should be a string.

Additionally, if we want to re-raise, is it necessary to use `:erlang.raise` as opposed to Elixir's `raise`?

Reply:
The catch could match any sort of error the user wishes. In this example we are just catching the raised `RuntimeError "oops"` above.

`:erlang.raise` is the same used by ErrorHandler here https://github.com/elixir-plug/plug/blob/v1.5.0/lib/plug/error_handler.ex#L98

`raise` wouldn't preserve the original stack trace but `reraise/3` would also probably work here

Reply:
Thanks for the explanation, @cschilbe :)